src/main/java/com/jagrosh/jdautilities/utils/SafeIdUtil.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2017 Kaidan Gustave
```

Comment:
I think that even though you created this file, it still needs to use the license of the repo